src/printer.js

```diff
@@ -2202,31 +2202,36 @@ function printNode(path, options, print) {
         path.call(print, "name"),
         node.curly ? "}" : ""
       ]);
-    case "static":
+    case "static": {
+      const printed = path.map(childPath => {
+        return print(childPath);
+      }, "items");
+
+      const hasValue = node.items.some(item => item.kind === "assign");
+
+      let firstVariable;
+
+      if (printed.length === 1) {
+        [firstVariable] = printed;
+      } else if (printed.length > 1) {
+        // Indent first var to comply with eslint one-var rule
```

Comment:
JS-specific comment, maybe remove "to comply with eslint one-var rule"

Reply:
Fixed :+1: